opendistro-elasticsearch-index-management.release-notes.md

```diff
@@ -1,3 +1,10 @@
+## Version 1.6.0.0 (2020-03-17)
+
+### New Features
+* Adds support for Elasticsearch 7.6.1 [PR #164](https://github.com/opendistro-for-elasticsearch/index-management/pull/164)
+* Add github action workflow for CI/CD [PR #161](https://github.com/opendistro-for-elasticsearch/index-management/pull/161) and release [PR #163](https://github.com/opendistro-for-elasticsearch/index-management/pull/163)
```

Comment:
These aren't added yet, they are still open PRs. Only add things that have been merged already (and the this open PR that you know will be merged).